### PR TITLE
Fixes #32918 - Host registration - Update packages

### PR DESCRIPTION
**Pull request by stejskalleos**
New parameter `update_packages` for update of
all packages on the host when registered

### CODE REVIEW

File: app/controllers/api/v2/registration_commands_controller.rb [hunk 1]
```diff
@@ -16,6 +16,7 @@ class RegistrationCommandsController < V2::BaseController
         param :jwt_expiration, :number, desc: N_("Expiration of the authorization token (in hours)")
         param :insecure, :bool, desc: N_("Enable insecure argument for the initial curl")
         param :packages, String, desc: N_("Packages to install on the host when registered. Can be set by `host_packages` parameter. Example: `pkg1 pkg2`")
+        param :update_packages, String, desc: N_("TODO")
```
Comment (ezr-ondrej) [inline app/controllers/api/v2/registration_commands_controller.rb]: ehem... xD

File: webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/UpdatePackages.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+const UpdatePackages = ({
+  updatePackages,
+  handleUpdatePackages,
+  isLoading,
+}) => (
+  <FormGroup fieldId="reg_update_packages">
+    <Checkbox
+      label={
+        <span>
+          {__('Update packages')} <LabelIcon text={__('TODO')} />
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/fields/UpdatePackages.js]: TODODODOOO :)

File: webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/__tests__/components/UpdatePackages.test.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import { testComponentSnapshotsWithFixtures } from '../../../../../common/testHelpers';
+
+import UpdatePackages from '../../components/fields/UpdatePackages';
+
+import { updatePackagesProps } from '../fixtures'
+
+describe('RegistrationCommandsPage fields - UpdatePackages', () => {
+  testComponentSnapshotsWithFixtures(UpdatePackages, { 'renders': updatePackagesProps });
```
Comment (ezr-ondrej) [inline webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/__tests__/components/UpdatePackages.test.js]: not sure that snapshot testing is the right choice here, onlything I can imagine worth testing here is that the onChange method gets called.

The snapshot just tests that we render `CheckBox` component within it :)
I believe there is more value in not having the test. Can't imagine this snapshot to discover any true positive failure :)

**Review by ezr-ondrej — COMMENTED** [review]
Reviewing just the React code and that seems fine, exept the testing (note my comment comes from the decision we want less snapshot testing going forward, we have plenty of tests like this in our codebase already) and few :trumpet: TODODOOOs around the code :stuck_out_tongue:

**Comment by stejskalleos** [comment]
@ezr-ondrej Rebased & Updated
TODOS removed, next time I need to check rebase result before push :/
And snapshot tests removed too.

**Review by ezr-ondrej — APPROVED** [review]
The code LGTM, I did not test, @ares if you do not get to testing it let me know, I can try, but would prefer if you could, since you have the setup already, right?

**Review by ares — APPROVED** [review]
Tested and works great! Thanks @stejskalleos and @ezr-ondrej, merging.

**Comment by ezr-ondrej** [comment]
CP 2.5: #8707

**Pull request merged**